azurelinuxagent/common/osutil/default.py

```diff
@@ -937,28 +937,23 @@ def get_endpoint_from_leases_path(pathglob):
         endpoint = None
 
         HEADER_LEASE = "lease"
-        HEADER_OPTION = "option unknown-245"
-        HEADER_DNS = "option domain-name-servers"
+        HEADER_OPTION_245 = "option unknown-245"
```

Comment:
OpenBSD referecnes HEADER_OPTION too.  Do any changes need to be made there?

Reply:
Yup, good catch. Will update there as well.

Reply:
After further discussion I will leave OpenBSD as is for now. Thanks for pointing this out.